conn.go

```diff
@@ -481,7 +480,7 @@ func (c *Conn) recv() error {
 	call, ok := c.calls[head.stream]
 	c.mu.RUnlock()
 	if call == nil || call.framer == nil || !ok {
-		log.Printf("gocql: received response for stream which has no handler: header=%v\n", head)
+		c.session.cfg.Log.Printf("gocql: received response for stream which has no handler: header=%v\n", head)
```

Comment:
It may be worth forwarding these methods so that `c.Log` or `c.session.Log` work just as well.